INSTALL.md

```diff
@@ -87,6 +96,27 @@ $ kubectl describe rs proxy
 
 If you want to allow images from the Docker Hub, take a look at the `--allow-docker-official` and `--allow-prefixes` arguments. This can be passed to Trow via the `trow.yaml` file.
 
+As an example: editing install/trow.yaml and adding the args:
```

Comment:
Maybe "The following example allows official images from Docker Hub and images with the prefix "registry.container-solutions.com/" to run in the cluster:"